related-resources.js

```diff
@@ -0,0 +1,49 @@
+/**!
+ * Copyright 2025 Digital Bazaar, Inc.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import * as Block from 'multiformats/block';
+import * as codec from 'multiformats/codecs/raw';
+import {fileTypeFromBuffer} from 'file-type';
+import fs from 'node:fs/promises';
+import {glob} from 'glob';
+import {sha256 as hasher} from 'multiformats/hashes/sha2';
+
+// Function to get digestMultibase for a given buffer
+export async function getDigestMultibase(buffer) {
+  const block = await Block.encode({value: buffer, codec, hasher});
+  const cid = await block.cid.toString();
+  return cid;
+}
+
+// Function to generate relatedResource object for an image file
+export async function generateRelatedResourceEntry(imagePath) {
+  // Read image file as buffer
+  const imageBuffer = await fs.readFile(imagePath);
+
+  // Encode digest using multibase (base58btc)
```

Comment:
I couldn't figure out how `base58btc` was specified anywhere here -- I guess it's a default or something with one of the imported APIs. So, just a note that I'm not diving too deeply in here to make sure this is doing what the comments say it's doing.

Reply:
Actually...that comment is a leftover. I switched to using the `sha256` hasher instead. See [line 12](https://github.com/credential-handler/vc-examples/pull/102/files/aacf6e85a4f3ca67b4a6fdf294fab0efe317ce4a#diff-64e48b3be8633ed1bcd14ba0f59d7c2bdbee826b369aa782abebae8846dad088R12). I'll clean-up that comment.